### ISSUE TITLE
add @asraa, @haydentherapper to sigstore security response team

### ISSUE THREAD

**Issue by bobcallaway** [sigstore/TSC#37]
Two new members were voted to join the sigstore security response team via email request to the TSC (copied below for transparency)

```
From: Trevor Rosen <trevrosen@github.com>
Date: Sun, 25 Sep 2022 09:43:26 -0500
Message-ID: <CAFUHT-PRtFnyZs+jfiDt0Jw+vTC9ZbzA5BBjQYh4rxGF=Cgs2w@mail.gmail.com>
Subject: Re: Proposing additional members for the security response team
To: Santiago Torres Arias <santiagotorres@purdue.edu>
Cc: Asra Ali <asraa@google.com>, Bob Callaway <bcallaway@google.com>, Dan Lorenc <dlorenc@chainguard.dev>, Hayden Blauzvern <hblauzvern@google.com>, Luke Hinds <lhinds@redhat.com>
Content-Type: multipart/alternative; boundary="000000000000b18fe705e9816fb2"

--000000000000b18fe705e9816fb2
Content-Type: text/plain; charset="UTF-8"

Lgtm

On Sat, Sep 24, 2022 at 1:58 PM Santiago Torres Arias <
santiagotorres@purdue.edu> wrote:

> lgtm
> On Sat, Sep 24, 2022 at 12:21:47PM -0400, Dan Lorenc wrote:
> > ---- External Email: Use caution with attachments, links, or sharing
> data ----
> >
> > Lgtm
> >
> > On Sat, Sep 24, 2022, 11:54 AM Bob Callaway <bcallaway@google.com
> <mailto:bcallaway@google.com>> wrote:
> > LGTM
> >
> > On Fri, Sep 23, 2022 at 12:07 PM Hayden Blauzvern <hblauzvern@google.com
> <mailto:hblauzvern@google.com>> wrote:
> > Hi TSC!
> >
> > tl;dr - I'd like to propose adding myself and Asra Ali to the security
> response team
> >
> > In order to make sure that there's appropriate coverage for responding
> to critical security reports and alerts triggered by the fuzzing
> infrastructure, I'd like to propose adding myself and Asra to the security
> response team. We've been involved with fixes for multiple CVEs and are
> very familiar with the infrastructure and clients, so we should be able to
> handle triage.
> >
> > As part of this, I'd also like to create a group for the team and update
> the documentation to reduce friction when reports are sent in. I'd also
> like to propose that we do not allow for encrypting reports, it adds
> unnecessary friction for verifiers. If this sounds good, let me know if you
> have a preference which email address should be added to the group. I'll
> default to the address I sent this to.
> >
> > Thanks,
> > Hayden
>
```
